chaos-monkey-spring-boot/src/test/java/de/codecentric/spring/boot/chaos/monkey/DefaultSettingsIntegrationTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2018 the original author or authors.
```

Comment:
Still not 100 % sure whether this should be 2020

Reply:
Its correct. I read in the internet that its the starting year the copyright is valid. If you change it to 2020 its not copyrighted in 2019 for example.

Source: https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated